cmd/state/internal/cmdtree/export.go

```diff
@@ -228,3 +229,41 @@ func newExportRuntimeCommand(prime *primer.Values) *captain.Command {
 
 	return cmd
 }
+
+func newExportBuildPlanCommand(prime *primer.Values) *captain.Command {
+	runner := export.NewBuildPlan(prime)
+	params := &export.BuildPlanParams{Namespace: &project.Namespaced{}}
+
+	cmd := captain.NewCommand(
+		"buildplan",
+		locale.Tl("export_buildplan_title", "Build Plan"),
```

Comment:
I believe the other export commands have titles like `Exporting <thing>`
```suggestion
		locale.Tl("export_buildplan_title", "Exporting Build Plan"),
```